composer.json

```
@@ -77,8 +86,26 @@
   ],
   "scripts": {
     "analyze": "vendor/bin/phpstan analyze --no-progress --memory-limit=1024M",
+    "behat": "behat",
     "format": "vendor/bin/phpcbf --report-summary --report-source .",
     "lint": "vendor/bin/phpcs --report-summary --report-source .",
-    "test": "vendor/bin/phpunit"
+    "phpunit": "vendor/bin/phpunit",
+    "test": [
+      "@analyze",
+      "@behat",
+      "@lint",
+      "@phpunit"
```

```suggestion
      "@behat",
      "@phpunit"
```

I don't think analyze and lint scripts should be part of tests.